src/components/Footer/Footer.js

```diff
@@ -0,0 +1,54 @@
+const Footer = () => {
+    return (
+      <footer class="footer d-flex direction-column align-center">
+        <p>
+          Made with{" "}
+          <span role="img" aria-label="Love">
+            ❤️
+          </span>{" "}
+          by Prakash Sakari
+        </p>
+        <div class="d-flex gap align-center padding-all-16">
+          <a
+            href="https://github.com/prakashsakari/"
+            class="link"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <img
+              class="source-image"
+              src="https://therightfit.netlify.app/assets/github-logo.png"
+              alt="GitHub"
+            />
```

Comment:
Make an array for your socials data and loop through it

Reply:
updated this, thanks